### ISSUE TITLE
Add ability to type in dates via an input field

### ISSUE THREAD

**Issue by jmpressman** [mercurymedia/elm-datetime-picker#12]

**Comment by aszenz** [comment]
I implemented this in a project recently by wrapping over this library, found it easy but also very app specific. The input field styles and the date format parsing were app specific.
Do you have a plan on how to make a generic api for this

**Comment by jmpressman** [comment]
Yes you are right that it is very app/region specific. We don't have any plans for a generic api at the moment due to the fact that our resources are currently focused on other projects.

That being said, an api for this would probably work in such a way that the user of the library would provide the date template/validations to be consumed by the library and triggered at the appropriate times. In other words, we would try to find a way to keep the library flexible across use cases by avoiding any sort of internal awareness of region.
